FGEGraphics/UISystem/UIGroup.cs

```diff
@@ -19,8 +19,5 @@ public class UIGroup : UIElement
 {
     /// <summary>Constructs a new group.</summary>
     /// <param name="pos">The position of the element.</param>
-    public UIGroup(UIPositionHelper pos) : base(pos)
-    {
-        Enabled = false;
-    }
+    public UIGroup(UIPositionHelper pos) : base(pos) => Enabled = false;
```

Comment:
doing a non-returning statement in a `=>` is cursed.
... Also doing `=>` for a constructor is cursed as a ctor is inherently nonreturning as well.

Nonfunctional functional lambdas are a strange construct